docs/services/gcp-services-guide.md

```diff
@@ -220,92 +200,20 @@ You are free to create more GCP `CloudsqlInstanceClass` instances to define more
-
-#### Portable Resource Classes
-
-Portable resource classes are used to define a class of service in a single
-namespace for an abstract service type. We want to define our GCP
-`CloudsqlInstanceClass` as the standard MySQL class of service in the namespace
-that our Wordpress resources will live in.
-
-* Define a `MySQLInstanceClass` in `mysql-standard.yaml` for namespace `app-project1-dev`:
-
-  ```bash
-  cat > mysql-standard.yaml <<EOF
-  ---
-  apiVersion: database.crossplane.io/v1alpha1
-  kind: MySQLInstanceClass
-  metadata:
-    name: mysql-standard
-  classRef:
-    kind: CloudsqlInstanceClass
-    apiVersion: database.gcp.crossplane.io/v1beta1
-    name: standard-cloudsql
-    namespace: $INFRA_NAMESPACE
-  EOF
-
-  kubectl -n app-project1-dev apply -f mysql-standard.yaml
-  ```
-
-* You can verify creation with the following command and output:
-
-  *Command*
-
-  ```bash
-  kubectl -n app-project1-dev get mysqlinstanceclasses
-  ```
-
-  *Output*
-
-  ```bash
-  NAME             AGE
-  mysql-standard   27s
-  ```
-
-Once again, you are free to create more `MySQLInstanceClass` instances in this
-namespace to define more classes of service. For instance, if you created
-`mysql-gcp-large` above, you may want to create a `MySQLInstanceClass` named
-`mysql-large` that references it. You may also choose to create MySQL resource
-classes for other non-GCP providers, and reference them for a class of service
-in the `app-project1-dev` namespace.
-
-> Portable classes are the resource classes that are not cloud specific and
-their purpose is to refer to cloud specific resource classes that will be
-used for provisioning. For example, MySQLInstanceClass is a portable class
-that can refer to CloudsqlInstanceClass (GCP) or RDSInstanceClass (AWS).
-
-You may specify *one* instance of a portable class kind as *default* in each
-namespace. This means that the portable resource class instance will be applied
-to claims that do not directly reference a portable class. If we wanted to make
-our `mysql-standard` instance the default `MySQLInstanceClass` for namespace
-`app-project1-dev`, we could do so by adding a label:
-
-```bash
-kubectl -n app-project1-dev label mysqlinstanceclass mysql-standard default="true"
-```
-For more details about resource classes in Crossplane, see
-[Default Resource Classes One-Pager](https://github.com/crossplaneio/crossplane/blob/master/design/one-pager-default-resource-class.md).
-
 #### Resource Claims
 
-Resource claims are used to create external resources by referencing a class of
-service in the claim's namespace. When a claim is created, Crossplane uses the
-referenced portable class to find a cloud-specific resource class to use as the
-configuration for the external resource. We need to create a claim to
-provision the MySQL database we will use in GCP.
+Resource claims are used to create external resources by being scheduled to a
+resource class and creating new managed resource or binding to an existing
```

Comment:
Suggest rewording this section for readability by a new user.

e.g.  Resource claims are used for dynamic provisioning of a managed resource (like a MySQL instance) by matching the claim to a resource class. This can be done in several ways: (a) rely on the default class marked  `resourceclass.crossplane.io/is-default-class: "true"`, (b) use a `claim.spec.classRef` to a specific class, or (c) match on class labels using a `claim.spec.classSelector`.

Note: claims may also be used in [static provisioning](link to quickstart guide) with a reference to an existing managed resource.